lib/matplotlib/ticker.py

```diff
@@ -1426,9 +1423,9 @@ def set_params(self, **kwargs):
         if 'integer' in kwargs:
             self._integer = kwargs['integer']
         if self._integer:
-            self._steps = [n for n in self._steps if divmod(n, 1)[1] < 0.001]
+            self._steps = [n for n in self._steps if _divmod(n, 1)[1] < 0.001]
 
-    def bin_boundaries(self, vmin, vmax):
```

Comment:
Good question.  Can we avoid that simply by renaming _raw_ticks back to bin_boundaries?